pkg/crc/ssh/keys_unix.go

```diff
@@ -32,13 +32,16 @@ func (kp *KeyPair) WriteToFile(privateKeyPath string, publicKeyPath string) erro
 		}
 
 		if _, err := f.Write(v.Value); err != nil {
+			_ = f.Close()
 			return ErrUnableToWriteFile
 		}
 
 		if err := f.Chmod(0600); err != nil {
+			_ = f.Close()
 			return err
 		}
 
+		_ = f.Close()
```

Comment:
These changes are unexpected in this commit, they probably should be moved to the next commit (" test: fix failing TestGenerateSSHKey on Windows ")

Reply:
Changes moved to next commit